guides/source/debugging_rails_applications.md

```diff
@@ -242,6 +242,33 @@ The contents of the block, and therefore the string interpolation, is only
 evaluated if debug is enabled. This performance savings is only really
 noticeable with large amounts of logging, but it's a good practice to employ.
 
+
+Debugging with the Web Console
+-------------------------------
+
+The web console allows you to create an interactive ruby session in your browser. An interactive 
+console is launched automatically in case on an error but can also be launched for debugging purposes
+by inserting `<%= console %>`anywhere in a view or `console` anywhere in a controller method.
+
+For example:
+
+```ruby
+# new.html.erb
+<%= console %>
+```
+
```

Comment:
@adipra3n Usually, it's a good idea to keep a single commit per PR. If they decide to merge, they will ask you to squash the commits first.